jumanji/environments/logic/rubiks_cube/viewer.py

```diff
@@ -70,15 +74,18 @@ def animate(
         ax = ax.flatten()
         plt.close(fig)
 
-        def make_frame(state_index: int) -> None:
-            state = states[state_index]
-            self._draw(ax, state)
+        images = self._draw(ax, states[0])
+
+        def make_frame(state: State) -> Sequence[Artist]:
+            for i, image in enumerate(images):
+                image.set_data(state.cube[i])
+            return images
```

Comment:
This was a bit confusing on first read, because I thought you somehow got all images from the `_draw` method, so I'd rather call it faces

```suggestion
        faces = self._draw(ax, states[0])

        def make_frame(state: State) -> Sequence[Artist]:
            for i, face in enumerate(faces):
                face.set_data(state.cube[i])
            return faces
```